tasks/oracle_job.yml

```diff
@@ -12,5 +12,6 @@
     storage_domain: "{{ cohesity_oracle.storage_domain | default('DefaultStorageDomain') }}"
     delete_backups: "{{ cohesity_oracle.delete_backups | default(False) }}"
     cancel_active: "{{ cohesity_oracle.cancel_active | default(False) }}"
-    time_zone: " {{ cohesity_oracle.time_zone | default('Asia/Kolkata') }}"
+    time_zone: "{{ cohesity_oracle.time_zone | default('Asia/Kolkata') }}"
```

Comment:
can you default this in the code to the system generated time where the ansible is run ? use pytz.

Reply:
We need to have installed pytz module for that. Can we make this package as a prerequisite?

Reply:
Oh , I thought pytz was native package, is there any other way to get this as default ?

Reply:
We have linux commands, we can run them.
datetimectl set <timezone>